src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -62,6 +63,16 @@ public function getPath()
         return $this->path;
     }
 
+    public function setLocales(array $locales)
+    {
+        $this->locales = $locales;
+    }
+
+    public function getLocales()
```

Comment:
array return type?

Reply:
I followed the rest of the methods here, none of them have return types or docblocks